src/System.AppContext/src/ApiCompatBaseline.net462.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.AppContext.TargetFrameworkName.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
I am going to file a bug for every new baseline file i create. Will do so for this too.